src/ConfigSetup.cpp

```diff
@@ -758,28 +758,43 @@ void ConfigSetup::Init(const char *fileName, MultiSim const*const& multisim)
       printf("%-40s %-4.4f \n", "Info: Rotation move frequency",
              sys.moves.rotate);
     } else if(CheckString(line[0], "IntraMEMC-1Freq")) {
-      sys.moves.intraMemc = stringtod(line[1]);
-      printf("%-40s %-4.4f \n", "Info: IntraMEMC-1 move frequency",
-             sys.moves.intraMemc);
-      if(sys.moves.intraMemc > 0.0) {
-        sys.intraMemcVal.enable = true;
-        sys.intraMemcVal.MEMC1 = true;
+      if(stringtod(line[1]) > 0.0){
+        if(sys.intraMemcVal.enable){
+          std::cout << "Error: Only one type of IntraMEMC move is supported at a time." << std::endl;
+          exit(EXIT_FAILURE);
```

Comment:
This is not the way I asked. We should not do error checking here. It's already done in verifyInputs() function. Just avoid overriding the value.